api/android/api/src/main/java/org/nnsuite/nnstreamer/NNStreamer.java

```diff
@@ -138,8 +140,8 @@ public static boolean isAvailable(NNFWType fw) {
 
         /* sub-plugin for given framework is available */
         if (available) {
-            String manufacturer = Build.MANUFACTURER.toLowerCase();
-            String hardware = Build.HARDWARE.toLowerCase();
+            String manufacturer = Build.MANUFACTURER.toLowerCase(Locale.getDefault());
```

Comment:
Is this just to fix Lint warning?

Reply:
yes, toLowerCase() is equivalent toLowerCase(Locale.getDefault())
```
String toLowerCase() {
  return toLowerCase(Locale.getDefault());
}
```